website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -16,6 +16,12 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### 1.10.x Agent Upgrade Notice
+
+Client agents must be Consul 1.10.5 or higher before attempting to upgrade
+servers to 1.11.x. There is a bug in Consul 1.10.0-1.10.4 which caused issues
+when communicating with 1.11.x servers.
+
```

Comment:
```suggestion
~> **Upgrading agents on 1.10.x**: Upgrade client agents to 1.10.5 or higher before attempting to upgrade servers to 1.11.x. A defect in Consul 1.10.0-1.10.4 may prevent client communication with 1.11.x servers.
```
This seemed more like information that should be called out in a box, rather than a specific topic.